### DISCUSSION TITLE
smart-splits cannot use with toggleterm and tmux

### DISCUSSION THREAD

**Discussion by gogongxt** [mrjones2014/smart-splits.nvim#199] [Q&A] [answered]
I open a split toggleterm on neovim, and right of the toggleterm is tmux.
Like the image shows.
![image](https://github.com/mrjones2014/smart-splits.nvim/assets/47113549/da5a2445-d4c3-4d44-816f-4420e6412359)

I cannot use ctrl+l to jump from toggleterm to tmux.
I can use smart-splits to jump between tmux apart from toggleterm.
Maybe i should add some config on toggleterm?
Thanks.

**Comment by mrjones2014** [comment]
Is `ctrl+l` bound in terminal mode? That could be the issue. Another possibility is that ctrl+l is bound as a shell keybind so its overriding the Neovim keybind.

**Comment by gogongxt** [comment, marked as answer]
Really thanks for your reply.
I check the keymap and find the keymap <c-l> in terminal.
So i change the following toggleterm config and solve the problem.
```lua
  if pcall(require, "smart-splits") then
    -- for smart-splits
    vim.keymap.set("t", "<C-h>", [[<cmd>lua require('smart-splits').move_cursor_left()<cr>]], opts)
    vim.keymap.set("t", "<C-j>", [[<cmd>lua require('smart-splits').move_cursor_down()<cr>]], opts)
    vim.keymap.set("t", "<C-k>", [[<cmd>lua require('smart-splits').move_cursor_up()<cr>]], opts)
    vim.keymap.set("t", "<C-l>", [[<cmd>lua require('smart-splits').move_cursor_right()<cr>]], opts)
  else
    vim.keymap.set("t", "<C-h>", [[<Cmd>wincmd h<CR>]], opts)
    vim.keymap.set("t", "<C-j>", [[<Cmd>wincmd j<CR>]], opts)
    vim.keymap.set("t", "<C-k>", [[<Cmd>wincmd k<CR>]], opts)
    vim.keymap.set("t", "<C-l>", [[<Cmd>wincmd l<CR>]], opts)
  end
```

**Reply by mrjones2014** [reply]
That makes sense, glad you figured it out!
